backoffice/modules/rating/components/RatingSearch.tsx

```diff
@@ -0,0 +1,84 @@
+import { Form, InputGroup } from 'react-bootstrap';
+import { RatingSearchForm } from '../models/RatingSearchForm';
+import { UseFormRegister } from 'react-hook-form';
+type Props = {
+  register: UseFormRegister<RatingSearchForm>;
+};
+
+const RatingSearch = ({ register }: Props) => {
+  return (
+    <>
+      <div className="row">
+        <div className="col-12 col-lg-6">
+          <div className="d-flex mb-4 w-75">
+            <Form.Label htmlFor="Created-From" className="label">
+              Created From:{' '}
+            </Form.Label>
+            <input
+              id="startDate"
+              className="form-control "
+              type="datetime-local"
+              {...register('createdFrom')}
+              defaultValue="2001-01-01T19:30"
+            />
+          </div>
```

Comment:
You can reuse component Input created before because it is generic for any react-hook-form.
Still valid for other inputs below.

Reply:
Yea I know, but your Input component doesnt fit my UI, so I customize it for mine. Thanks for your suggest!